src/modules/fancyzones/lib/ZoneWindow.cpp

```diff
@@ -446,7 +446,7 @@ ZoneWindow::MoveWindowIntoZoneByIndexSet(HWND window, const std::vector<int>& in
 {
     if (m_activeZoneSet)
     {
-        m_activeZoneSet->MoveWindowIntoZoneByIndexSet(window, m_window.get(), indexSet, false);
+        m_activeZoneSet->MoveWindowIntoZoneByIndexSet(window, m_window.get(), indexSet);

```

Comment:
Why was it false in this case?

Reply:
@ivan100sic any idea about this?

Reply:
Probably by mistake, I believe it was actually the cause of #3111